tests/meta/upload/test_meta_upload_scoutapi.py

```diff
@@ -8,6 +8,17 @@
 from cg.meta.upload.scoutapi import UploadScoutAPI
 from cg.store import Store
 
+CASE_FILE_PATHS = ["multiqc"]
+
+RESULT_KEYS = [
+    "family",
```

Comment:
Family?

Reply:
Yes, I think the mock analysis is using the family key and I do not want to change it in this PR as it is used by many tests.

Reply:
seems like a scout thing, might be beyond scope of this PR